### DISCUSSION TITLE
Github Action for Upgrading - Nice to have

### DISCUSSION THREAD

**Discussion by meseguera** [directus/directus#10520] [Feature Requests]
Is there a directus Github Action to upgrade minor versions, to v9.2.2 from v9.2.1 
One could setup a self-hosted runner (I did- not hard) to perform the upgrade.  
Thank you!

Alex Meseguer

**Comment by rijkvanzanten** [comment]
To update it within your own project? I'm not sure how to trigger a GH action based on activity in another repo, but that being said, doing an automated upgrade should be as easy as

```
npm install directus@9.2.2 --save-exact
directus database:migrate latest
```

(though you might want to work in some way to backup your database/files before starting just in case)
